frontend/src/App.tsx

```diff
@@ -1,7 +1,22 @@
 import React from 'react';
+import { CssBaseline, Typography } from '@material-ui/core';
+import { BrowserRouter as Router, Route, Routes, Link } from 'react-router-dom';
+import { SignInContainer, signIn } from './AppReceiver';
 
 function App() {
-  return <></>;
+  return (
+    <>
+      <CssBaseline />
+      <Router>
+        <Link to={signIn} style={{ width: '100%', textAlign: 'center', textDecoration: 'none' }}>
```

Comment:
Please move all the styles inside separate file

Reply:
Its only test link for demo.  In 003 we will replace this Link components.

Reply:
But could you please move all the styles of it to the styled file?

Reply:
Yup, Sure